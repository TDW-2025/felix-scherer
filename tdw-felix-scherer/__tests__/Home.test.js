import React from "react";
import { render } from "@testing-library/react";
import Home from "@/app/page";

describe("Home Component", () => {
  it("matches the snapshot", () => {
    const { asFragment } = render(<Home />);
    expect(asFragment()).toMatchSnapshot();
  });
});