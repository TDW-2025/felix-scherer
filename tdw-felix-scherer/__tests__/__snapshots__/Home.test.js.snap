// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Home Component matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="page"
  >
    <h1>
      Hello World!
    </h1>
    <h2>
      This is the edited page.
    </h2>
    <div>
      Let's test the full pipeline!
    </div>
    <li>
      <ul>
        Lint
      </ul>
      <ul>
        Prettify
      </ul>
      <ul>
        Test
      </ul>
    </li>
    <a
      href="/array-map"
    >
      <button>
        Navigate
      </button>
    </a>
  </div>
</DocumentFragment>
`;
